### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.1.1][IntelliJ][Wipro]: On moving the "Artifact" with "Up and down" buttons in Run\debug configurations dialog , the entire dialog becomes not accessible.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1509]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Create a new project.
3. Tab to and select "edit configurations" present at right side of the dialog.
4. "Run/Debug Configurations" dialog will open.
5. Tab to and select the "Up\down" button, to move the artifact,  then observe the keyboard behaviour.

**Note:** Same issue is observed after deleting the artifact under "Before launch" expand collapse button.

**Expected Result:**
After moving the Artifact the keyboard focus should stay on Up\down  button and should be able to tab with tab key.

**Actual Result:**
No keyboard action is taking after moving the "Artifact"  Up and down in Run\debug configurations dialog.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)

![edit config](https://user-images.githubusercontent.com/22956130/36149245-e2ceb8e6-1073-11e8-9d33-d6baeb15e9ed.PNG)

**Comment by Eskibear** [comment]
The scenario has nothing to do with the toolkit.

**Comment by Eskibear** [comment]
Moreover, the UI is also determined by IntelliJ IDEA, it's out of the toolkit's scope.

**Comment by Eskibear** [comment]
Reference: https://youtrack.jetbrains.com/issue/IDEA-191322

Tracked in #1627

**Comment by Eskibear** [comment]
As I mentioned long before, this use case has nothing to do with our product. 

* **Importance of experience**:  minimal. 
* **Quality of experience**: not impacted.
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749986
